gtsam/geometry/Rot3.h

```diff
@@ -435,27 +440,18 @@ namespace gtsam {
 
     /**
      * Accessor to get to component of angle representations
-     * NOTE: these are not efficient to get to multiple separate parts,
-     * you should instead use xyz() or ypr()
-     * TODO: make this more efficient
      */
-    inline double roll() const  { return ypr()(2); }
+    inline double roll() const  { return xyz()(0); }
```

Comment:
TODO of making more efficient is gone, but *is this more efficient?

Reply:
The TODO note said to use either `ypr` or `xyz`

```
* NOTE: these are not efficient to get to multiple separate parts,
* you should instead use xyz() or ypr()
* TODO: make this more efficient
```

I figured it was a leftover comment from a previous commit (similar to the boost TODO for rng). `ypr` was internally calling `xyz` so all I did was remove the extra layer. Internally it is still calling the `RQ` algorithm.

Reply:
So, add the TODO again :-)